news/2019-05-06-project-loved-week-of-may-5th.md

```diff
@@ -0,0 +1,229 @@
+
+### Navigation
+
+- [osu!](#osu)
+- [osu!taiko](#osutaiko)
+- [osu!catch](#osucatch)
+- [osu!mania](#osumania)
+
+## <a name="osu" id="osu"></a>osu!
+
+osu! Loved candidates were chosen by [DigitalHypno](https://osu.ppy.sh/users/4384207), [-Roxas](https://osu.ppy.sh/users/1986262), [BeasttrollMC](https://osu.ppy.sh/users/3171691), [Bubbleman](https://osu.ppy.sh/users/5182050), [fartownik](https://osu.ppy.sh/users/56917) and [waywern2012](https://osu.ppy.sh/users/5870453)!
+
+---
+
+[![](/wiki/shared/news/2019-05-06-project-loved-week-of-may-5th/osu/1-internet-bitch-p-light-remix.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[DJ'TEKINA//SOMETHING - Internet bitch P\*Light Remix](https://osu.ppy.sh/beatmapsets/357487#osu) by [Meg](https://osu.ppy.sh/users/1872496)  
+175 BPM, 3:42 | 5.86★  
+*description written by [fartownik](https://osu.ppy.sh/users/56917)*
+
+"Internet lady" (as it's called in the hood) is the Meg's most popular map in the graveyard, reaching almost 200 favourites and surely not without a reason. The rhythm that Meg chooses here is simply excellent, focusing on circle-only doubles and quads in some sections to smoothly transform into cut streams in the more intense ones. The map is truly a test of finger control and is one of the most fun maps of its kind.
+
+[![](/wiki/shared/news/2019-05-06-project-loved-week-of-may-5th/osu/2-onlywonder.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[frederic - ONLYWONDER](https://osu.ppy.sh/beatmapsets/807885#osu) by [n0ah](https://osu.ppy.sh/users/3086393)  
+168 BPM, 4:35 | 5.75★, 5.77★  
+*description written by [fartownik](https://osu.ppy.sh/users/56917)*
+
+ONLYWONDER is the last remaining piece without the leaderboard of n0ah's frederic map collection. This collection features four different frederic songs mapped in a very old school, gimmicky fashion. The set, similarly to all the other maps from the series, comes with two variants: an AR5, for the hardcore readers, and an AR8 for scrubs like myself. The map itself focuses on hard-to-read patterns, using weird stream angles and anti-jumps to achieve the goal, all decorated with a semi-strict, grid-style note placement. n0ah himself says this is probably the worst out of his frederic maps, but I think otherwise—it's arguably the most fun one of them all.
+
+[![](/wiki/shared/news/2019-05-06-project-loved-week-of-may-5th/osu/3-ryuu-to-shoujo-to-decoherence.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[Kuroneko Dungeon - Ryuu to Shoujo to Decoherence](https://osu.ppy.sh/beatmapsets/94671#osu) by [ignorethis](https://osu.ppy.sh/users/27343)  
+165 BPM, 1:57 | 7.08★  
+The [Hyper] and [Taiko Oni] difficulties are *not* being nominated for Loved.  
```

Comment:
```suggestion
The [Hyper] and [Taiko Oni] difficulties will *not* be nominated for Loved.  
```

Reply:
this news post is nominating them tho so i think present tense is right?

Reply:
is using `be` in gerund form correct though?

Reply:
yeah? maybe? i don't even know the rule but this sounds normal to me